cli/src/commands/expirationExports.js

```diff
@@ -0,0 +1,51 @@
+import Download from 'lib/models/download';
+import SiteSettings from 'lib/models/siteSettings';
+import { SITE_SETTINGS_ID } from 'lib/constants/siteSettings';
+import { get } from 'lodash';
+import moment from 'moment';
+import { map } from 'bluebird';
+import Organisation from 'lib/models/organisation';
+
+const expireExportsForOrganisation = async (organisation) => {
+  const todayDate = moment();
+  // const cutOffDate = todayDate.subtract(ttl, 'seconds');
+
+  const exports = await Download.find({
```

Comment:
> Why aren't we removing the downloads right here instead of mapping over each one and removing them? Removing them here would be quicker.

https://github.com/LearningLocker/learninglocker/pull/1215/commits/1b1750a5fc8d720ebb9dec00fa7260e58f87c354 fixed

> Also we can do a $in on the organisation to remove all the downloads at once, rather than mapping over orgs too.

https://github.com/LearningLocker/learninglocker/pull/1215/commits/abe4dc8ada44e04a6bef14e05e9cec3935faa75b fixed

Reply:
Awesome thanks @h-kanazawa 👍